fastify.d.ts

```diff
@@ -286,6 +286,14 @@ declare namespace fastify {
      */
     register<T extends RegisterOptions<HttpServer, HttpRequest, HttpResponse>>(plugin: Plugin<HttpServer, HttpRequest, HttpResponse, T>, opts?: T): FastifyInstance<HttpServer, HttpRequest, HttpResponse>
 
+    /**
+     * `Register a callback that will be executed just after a register.
+     * It can take up to three parameters
+     */
+    after(afterListener?: (err: Error) => void): void
+    after(afterListener?: (err: Error, done: Function) => void): void
+    after(afterListener?: (err: Error, context: FastifyInstance<HttpServer, HttpRequest, HttpResponse>, done: Function) => void): void
```

Comment:
If the `afterListener` callback is optional there should be a test for that. I think it's required though so the `?` would need to be removed.